SPEC.md

```diff
@@ -313,3 +313,7 @@ The test index makes use of symbolic language in describing connection and messa
     The node should be able to accept connections after such a request.
     These tests attempt a handshake with long 'User-Agent'/'Server' headers and ensures that the connection
     is rejected. Then, it attempts a normal connection and ensures that the connection is established.
+
+### ZG-RESISTANCE-002
+
+    The node should disconnect when it receives random bytes instead of an actual message
```

Comment:
I'd love to see more text here. Is it before/after handshake? How long are the random bits. Perhaps touch on why `rand_chacha` was needed?

Reply:
done